### ISSUE TITLE
split data into training and test set

### ISSUE THREAD

**Issue by plpabla** [plpabla/krk_prices#37]
We need a script to split data into training and test set before we do any preprocessing.

It makes sense to make equal number of samples for each district (`train_test_split(..., stratify=df['district'])`)
